Okta.Sdk.Abstractions/DefaultHttpClient.cs

```diff
@@ -32,7 +32,7 @@ public static class DefaultHttpClient
                 AllowAutoRedirect = false,
             };
 
-            if (proxyConfiguration != null)
+            if (proxyConfiguration != null && !string.IsNullOrEmpty(proxyConfiguration.Host))
```

Comment:
While testing locally I found my .yaml config with `null` for the proxy values was interpreted as a proxy with host `""`.  Since an empty string is not the same as null it was attempting to use that and failing.  This change ensures that the config is not null and the actual host setting has some value.